### PR TITLE
fix typo `documenation`

### PR DESCRIPTION
**Pull request by Kvicii**

### CODE REVIEW

File: src/doc/rustdoc/src/references.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ If you know of other great resources, please submit a pull request!
 - [Github tagged RFCs]
 - [Github tagged issues]
 - [RFC (stalled) front page styleguide]
-- [Guide on how to write documenation for a Rust crate]
+- [Guide on how to write documentation for a Rust crate]
```
[the remark this answers is not in the document]
Comment (Kvicii) [reply src/doc/rustdoc/src/references.md:19]: what？

[the remark this answers is not in the document]
Comment (notriddle) [reply src/doc/rustdoc/src/references.md:19]: On line 25, there's the same typo.

[the remark this answers is not in the document]
Comment (Kvicii) [reply src/doc/rustdoc/src/references.md:19]: @GuillaumeGomez @notriddle 
hi, I fixed. PTAL

**Review by notriddle — APPROVED** [review]

**Pull request merged**